Cargo.toml

```diff
@@ -10,3 +10,7 @@ members = [
     "zbus_xmlgen",
 ]
 resolver = "2"
+
+[workspace.lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(tokio_unstable)'] }
```

Comment:
would be nice to have more detail in the commit log, especially what `check-cfg` is and why it needs to say `tokio_unstable`.

Reply:
ok